platforms/posix/src/lockstep_scheduler/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ if(NOT PROJECT_NAME STREQUAL "px4")
 
     set (CMAKE_CXX_STANDARD 11)
 
+	add_definitions(-DUNIT_TESTS)
```

Comment:
We should probably move the whole file to tabs.

Reply:
Right, I didn't realize. Commit pushed.

Reply:
Quick comment because I just noticed this (not PR related). The CMakeLists.txt doesn't need to be different for inclusion in px4 or building standalone.

Reply:
It does because it can be used for standalone unit tests. And I would like to keep these tests until we have figured out a way to port them over into PX4.